tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,22 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should return an error if trying to delete a file that does not exist', function(done) {
+    var fsPromises = util.fs().promises;
+
+    fsPromises.unlink('/myFile')
+      .then(result => expect(result).not.to.exist)
+      .catch(error => {
+        expect(error).to.exist;
+        expect(error.code).to.equal('ENOENT');
+      })
+      .then(() => done());
```

Comment:
To make this code more promise-friendly, please remove this line as `done()` is no longer needed.

Reply:
thank you for your suggestion :)